src/System.Threading.Thread/tests/System.Threading.Thread.Tests.builds

```diff
@@ -2,7 +2,12 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Threading.Thread.Tests.csproj" />
+    <Project Include="System.Threading.Thread.Tests.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+    </Project>
+    <Project Include="System.Threading.Thread.Tests.csproj">
+      <OSGroup>Unix</OSGroup>
+    </Project>
```

Comment:
I'm not sure I understand why you need this if you don't have tests that are specific to Windows or Unix. It looks like the test assembly is exactly the same in both cases, so I'm not sure if this change is needed.